rules.bzl

```diff
@@ -22,7 +22,8 @@ def lombok_java_library(name, srcs, deps = [], lombok_jar = DEFAULT_LOMBOK_JAR,
     if debug:
         cmd.extend(["pwd", "set -x"])
 
-    cmd.append("TMP=$$(mktemp -d -p . || mktemp -d -t bazel-tmp -p .)")
```

Comment:
I tried running GNU temp with the second command after pipe and it complained about the template that it does not contain X characters to be replaced with random strings

```
$ gmktemp -d -t bazel-tmp -p .
gmktemp: too few X's in template ‘bazel-tmp’
```

is it possible that this command is redundant?

Reply:
Indeed I wrote all these code over 18 months ago when I was barely starting to grasp how bazel works and how rules writing works, given the rule is already inside a sandbox we could just do our whole processing inside some relative path, no need to go to /tmp or whatever, could you make the changes?

Also since I think bazel 5 you can declare_directory, and we could make that our output place as long as we return that with a depset on DefaultInformation.

Can you at least apply the first part of changes? I'm not doing java lately making it hard for me to dedicate even 10 minutes to this topic, otherwise I'll fix it during BazelCon if I get the time

Reply:
I believe the two lines bellow do the thing - the dot in `TMPDIR=.` says to create a temporary folder within the current sandbox.

Reply:
sure, but we don't need it anyway, my idea around mktempd was to not get collisions with names